recipes/libressl/2.9.2/conanfile.py

```diff
@@ -0,0 +1,73 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class LibreSSLConan(ConanFile):
+    name = "libressl"
+    version = "2.9.2"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.libressl.org/"
+    license = "OpenSSL - OpenBSD"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = "shared=False", "fPIC=True"
+    topics = ("SSL", "TLS", "openssl")
+    description = "LibreSSL is a version of the TLS/crypto stack forked from OpenSSL in 2014, with goals of modernizing the codebase, improving security, and applying best practice development processes."
+    generators = "cmake"
+
+    @property
+    def libressl_src(self):
+        return "%s-%s" % (self.name, self.version)
+
+    def config_options(self):
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+
+    def build(self):
+        # needed to properly overwrite MSVC runtime
+        tools.replace_in_file(os.path.join(self.libressl_src, "CMakeLists.txt"), "project (LibreSSL C ASM)", """project (LibreSSL C ASM)
```

Comment:
why you don't create a CMake wrapper instead? You can obtain it from other Conan package in CCI.

Reply:
I took a look at a few other packages with no success, do you have a specific package in mind? So that I can avoid looking at random :stuck_out_tongue: